### PR TITLE
BMP280 based Arduino 

### PR DESCRIPTION
**Pull request by yutarour**
## Describe your project in 1-3 sentences

I am going to build a enclosure for a BMP280 based Arduino temperature, humidity and pressure logger. This also involves making the logger.

## Hack Club Slack handle
Yutaro U.

## Checklist before submitting proposal:

- [✔️] I am a current high school student (or younger).
- [✔️] I have submitted an RSVP at <https://hackclub.com/winter>
- [✔️] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [✔️] I have self-reviewed my project proposal.

### CODE REVIEW

**Review by yutarour — COMMENTED** [review]
Modified the budget by removing the resin which will be paid out of pocket in order to accommodate sales tax. Shipping is free due to amazon prime. @bellesea

File: yutarour.md [hunk 1]
```diff
@@ -61,5 +61,7 @@ I will be making a model of the enclosure in fusion 360 and printing it with a r
 | Isopropyl Alchohol        | Already Own                           | $0.00  |
 | 9V battery                | Already Own                           | $0.00  |
 | Female to Male wires      | Already Own                           | $0.00  |
-| Translucent Resin         | https://a.co/d/cN24vPx                | $17.99 |
-| Total                     |                                       | $248.98|
+| Translucent Resin         | Will purchase with own money          | $17.99 |
+| Tax (6.5%)                 |                                       | $15.01|
```
Comment (yutarour) [inline yutarour.md:65]: Sales tax: 6.5%

**Review by yutarour — COMMENTED** [review]
Updated BOM so that tax is incorporated into the total.
Merged the commits from last time.

File: yutarour.md (hunk 1 above)
Comment (yutarour) [reply yutarour.md:65]: Amazon has already included the sales tax in their listing, but nothing bad about being safe with the budget

File: yutarour.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+| ELEGOO Mercury 2 in 1 Washing and curing machine     | https://a.co/d/60ervD3                | $111.99|
+| Arduino Uno               | Already Own                           | $0.00  |
+| BMP280 sensor             | Already Own                           | $0.00  |
+| Isopropyl Alchohol        | Already Own                           | $0.00  |
+| 9V battery                | Already Own                           | $0.00  |
+| Female to Male wires      | Already Own                           | $0.00  |
+| Translucent Resin         | Will purchase with own money          | $17.99 |
+| Tax (6.5%)                 |                                       | $15.01|
+| Shipping                  |Free with Amazon Prime                 | $0.00  |
+| Total                     |                                       | $246.00|
```
Comment (bellesea) [inline yutarour.md:67]:
```suggestion
| Total                     |                                       | $263.99|
```

**Review by bellesea — APPROVED** [review]
@yutarour thanks sm for so quickly adding the changes requested! it's ok if it goes over budget, we'd just cover up to $250 :) 

@MelSmith104 will reach out with next steps within 48 hours. Thanks for your patience and can't wait to see you work on this project :)

**Comment by yutarour** [comment]
Sounds awesome, can't wait to build!

**Review by MelSmith104 — COMMENTED** [review]
hey @yutarour ! i love your project idea! our next step is to start the process of getting you set up on hack club bank! i'm dm'ing you on slack right now!

**Pull request merged**